### DISCUSSION TITLE
SCP-009

### DISCUSSION THREAD

**Discussion by Doctor-Bright** [share121/inter-knot#2973] [其他]
# 项目编号：SCP-009

## 项目等级：Euclid

***

# 回收之前的SCP-009（图片文件）
![百度](http://scp-wiki.wdfiles.com/local--files/scp-009/009.png)

***

## 特殊收容措施：

项目应收容于一尺寸不小于2m x 2m x 2m的隔热合金密闭储存水槽中。

在未进行实验的任何情况下，SCP-009都不允许被暴露于超过0°C的温度下，且项目的收容区域的30米内不得有水基溶液。项目的收容间应有必须随时监控着的温度传感器，且应有不少于三（3）台的制冷机保持制冷。任何温度传感器或制冷系统的故障需要被汇报并立即维修。若收容区域中的温度上升到-5°C以上，收容间应被封锁并注满冷却剂，直到温度回到安全程度（-25°C至-30°C）。

于实验期间，收容区域应保持完全真空，且人员在接触SCP-009时必须穿着全套环境防护用具。实验之后，所有装备、人员以及其余工具必须进行脱水程序并且实行不少于12小时的隔离。任何发现展现SCP-009特性的水分应尽快被隔离并加至收容区域中。发现被SCP-009污染的活体生物应由[数据删除]做远程处决，提取出的SCP-009分子应回归收容。

描述：
>SCP-009为大约███公升的物质，外观除明显的亮红色以外与蒸馏水（H2O）类似。此种红色在所有的相都可识别，并且是在其异常特性表现之前最合适的辨识污染方法。与正常的水相比，SCP-009于-100°C至0°C的温度间呈现液态，在更高的温度则为固态。在低于-100°C的温度时，SCP-009蒸发为类似蒸汽的气态。

>对SCP-009原子结构的调查没有定论。该物质似乎与水分子一致，除了与标准的焓变定律不同的[已编辑]。Site ██的常驻异空间物理学专家Dr. █████表示SCP-009可能来自于一个有着不同物理定律的宇宙。

>然而，SCP-009最危险的特性是其能污染正常的H2O。当与任何水基溶液接触时，SCP-009将以未知机制把异常特性转移至其他物体和生物上。实验表明其能同化冰、蒸汽、茶、果汁、海水、血液以及[已编辑]。此发生变化的过程耗费时间取决于温度和受影响物质的化学成分，并且观测到需耗时3分至██小时。

***

在D级人员上的实验表明了物质转化的过程，已经发现遵循一致模式：

1. 最初暴露：实验对象暴露于SCP-009，其开始同化任何位于所接触表面上的水分。此阶段的生物通常不会注意到任何异常症状，除了轻微的暖和感。

2. 表层转化：因对象和SCP-009本身产生的热量温度升高至0°C，SCP-009开始在接触区域成霜。此阶段可耗时一（1）分至█小时，对象在此期间开始从表皮感觉到[数据删除]晶体。

3. 深层组织转化：SCP-009温度指数增长，导致对象身体各处的反应失控，结果[数据删除]冰晶经由多重器官衰竭扩散[已编辑]。由于冰晶[数据删除]，实际失血极微，使对象可维持存活并有意识达██小时。

4. >[数据删除]

在D级人员上的实验于4/23/20██起停止。

***

*附录：对象被回收时的情况：对象于11/05/19██在阿拉斯加的████被发现。基金会在接到了当地居民████ Tribe的报告后介入，他穿过了村庄██千米外的一艘看似发生海难的捕杀海豹的船只上面的破碎的尸体*

*所有受害者都被红色的冰包裹。死亡的原因被记录为内出血，但最近实验发现[已编辑]。据此推测较低的环境温度减缓了结冰的过程。这延长了转化时间至██小时，并使受害者保持意识清醒直到[数据删除]*

**当前未知SCP-009的起源。正对类似的事件或该区域的材料进行调查。现场证据显示[数据删除]，可能涉及了SCP-███**
***

见[探索记录A009-1](https://scp-wiki-cn.wikidot.com/exploration-log-a009-1)获取细节。

***
附录：11/09/19██
于初步汇报以及实例检索后，证实由MTF-B7寻获的蛛状实体（见[附件](https://scp-wiki-cn.wikidot.com/exploration-log-a009-1)）确实为先前[SCP-3023](https://scp-wiki-cn.wikidot.com/scp-3023)的未知实体。调查已揭露实体起源于由于[数据删除]而导致的[已编辑]。

附录：12/06/19██
在反复调查后，应注意到根据最初受害者被发现时的海岸线部分是距离海滨约█米的贫瘠岩石，且足够干燥及寒冷以避免周围区域的大规模污染。要是该处更接近水的话，毫无疑问会引起一场灭绝级别的事件。

<u>将SCP-009升至Keter级别的考虑正检查中</u>

附录：12/16/20██

在另行通知之前，禁止对SCP-009进行超低温实验。人员只能对可控量的对象使用液态氮，并只能在温度达到可接受的水平时。

相关笔记：SCP-009在冷核聚变研究中可能的应用有待评估。

O-5指挥部备忘： 1/09/20██
我们决定暂时维持这东西为Euclid级。我们了解那提出的顾虑，但只要你保持通电且无人靠近其收容区域，便不会有问题。毕竟这就是为何我们将它保存在Site ██的原因。

**至于冷核聚变研究，我们正要谈谈。坦白说，我们没有足够预算来处理另一个如同Site ██的[已编辑]般的混乱情况。营救团队仍未找到Dr. █████的[数据删除]**

***

>+交叉测试报告 {需提供3级权限}

Secure.Contain.Protect

>-审核:[已编辑]

**Comment by LShanShi** [comment]
这个委托我接了!

**Reply by LShanShi** [reply]
这个委托我接...不了!

**Comment by cpu644** [comment]
这个委托我一点都接不了。

**Comment by chenx198** [comment]
我将化身满级造反DD！

**Comment by Pheathon** [comment]
这个委托，我……接不了一点。。。

**Comment by opqphp** [comment]
这个委托我怕了<img src="https://bbs-static.miyoushe.com/static/2023/11/17/e0564870de2f3d1c39dfc41a8dcd9d2b_383241188874318913.png" alt="芙宁娜-好困难" width="75" >

**Comment by 1ctrl-cv** [comment]
这个委托我接了

**Comment by Samu1KU** [comment]
给我干哪来了，这还是新艾利都吗

**Comment by 66263184** [comment]
绳网不大，创造神话。
